### PR TITLE
COL-395, add asset view as an activity

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/COL-395

I'm setting up Canvas to verify locally. In the meantime, @nicolaasmatthijs can review diff below. I assume changes in daily email are TBD. 

Fyi, follow up task: https://jira.ets.berkeley.edu/jira/browse/COL-408, "Add views to the Engagement Index configuration so instructors can chose to award points for views"

### CODE REVIEW

**Comment by johncrossman** [comment]
@nicolaasmatthijs amendment is in. Travis may need some love... Checking.

File: node_modules/col-assets/lib/api.js [hunk 1]
```diff
@@ -208,8 +209,14 @@ var incrementViewsIfRequired = function(asset, incrementViews, callback) {
       log.error({'err': err, 'id': id}, 'Failed to increase the views on an asset');
       return callback({'code': 500, 'msg': err.message});
     }
+    ActivitiesAPI.createActivity(ctx.course, ctx.user, 'view_asset', id, CollabosphereConstants.ACTIVITY.OBJECT_TYPES.ASSET, null, null, function(err) {
+      if (err) {
+        log.error({'user': ctx.user, 'err': err}, 'Failed to create activity \'view_asset\' for user');
+        return errorCallback(err);
```
Comment (nicolaasmatthijs) [inline node_modules/col-assets/lib/api.js]: `errorCallback` doesn't appear to be defined anywhere. Should just be `return callback(err);`.

File: node_modules/col-assets/lib/api.js [hunk 2]
```diff
@@ -208,8 +209,14 @@ var incrementViewsIfRequired = function(asset, incrementViews, callback) {
       log.error({'err': err, 'id': id}, 'Failed to increase the views on an asset');
       return callback({'code': 500, 'msg': err.message});
     }
+    ActivitiesAPI.createActivity(ctx.course, ctx.user, 'view_asset', id, CollabosphereConstants.ACTIVITY.OBJECT_TYPES.ASSET, null, null, function(err) {
```
Comment (nicolaasmatthijs) [inline node_modules/col-assets/lib/api.js]: Can you add a test that verifies that an asset view activity doesn't contribute to the engagement index score by default?

**Comment by nicolaasmatthijs** [comment]
Finished review. Back to @johncrossman

File: node_modules/col-assets/lib/api.js (hunk 2 above)
Comment (johncrossman) [inline node_modules/col-assets/lib/api.js]: @nicolaasmatthijs Yes. Thanks

**Pull request closed without merging**